logos/tests/unit/test_terminal_logging.py

```
@@ -0,0 +1,76 @@
+"""Unit tests for terminal_logging helpers: format_bytes and format_number."""
+
+import pytest
```

Flake8 (7.3.0)

[error] 'pytest' imported but unused

(F401)